S/SLATEC/build_tarballs.jl

```diff
@@ -76,8 +91,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="LAPACK_jll", uuid="51474c39-65e3-53ba-86ba-03b1b862ec14")),
-    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363")),
+    Dependency(PackageSpec(name="libblastrampoline_jll")),
```

Comment:
Hi,
Thanks for the comments. I did the update, because the current version no longer build with Julia 1.9.
So no problem with updating the version and adding the compat to 1.9.

Reply:
> you really ***have*** to specify the compat with it.

```suggestion
    Dependency(PackageSpec(name="libblastrampoline_jll"); compat="5.4"),
```